utils/queryApiGateway.ts

```diff
@@ -112,6 +113,51 @@ export function getUniqueNumber(req: IncomingMessage): string {
   return req.headers[idHeaderName.toLowerCase()] as string
 }
 
+/**
+ * Check if the API responded with a null response body
+ *
+ * We do not expect any of these in production, in theory.
+ */
+export function reponseIsNull(apiBody: Claim): boolean {
+  const response: Claim = JSON.parse(JSON.stringify(apiBody)) as Claim
+  delete response.uniqueNumber
+
+  // This is the response for a bad UniqueNumber, or a UniqueNumber without a claim
+  const nullResponseShort = {
+    claimDetails: null,
+    hasCertificationWeeksAvailable: false,
+    hasPendingWeeks: false,
+    pendingDetermination: null,
+  }
```

Comment:
If you include this, you are in effect making it so that this is unreachable: https://github.com/cagov/ui-claim-tracker/blob/d79ff60b15969589f2e9da3f7342d1d82ecbd1e6/utils/queryApiGateway.ts#L183-L190

Reply:
Therefore, I think you should either not include the `nullResponseShort` or handle the mismatchError first before you handle the new null response error.

Reply:
oh interesting - I was considering `mismatched UniqueNumber` when the API gave us a `null` to be incidentally catching the `null` response case, and that this is a more accurate error for that situation. That is still reachable if we get back the different unique number, or get back some claim data but a null unique number.

Reply:
Oh interesting. I see what you mean. This sounds good to me!